### PR TITLE
Volume script device name fallback

### PR DESCRIPTION
**Pull request by bryan-bar**
Changes:
* AWS volume script updated to allow fallback to device name instead of `nvme` tool
  * Device is still mounted with its `UUID` inside of `/etc/fstab`
    * Users need to update `/etc/fstab` if they reformat the device
* GCloud and Azure updated to match where possible

### CODE REVIEW

File: edbterraform/data/terraform/gcloud/modules/machine/setup_volume.sh [hunk 1]
```diff
@@ -1,20 +1,66 @@
...
-echo "UUID=${DEVICE_UUID} ${MOUNTPOINT} xfs noatime 0 0" | sudo tee -a /etc/fstab
+# Mount point and volume creation
+sudo mkfs.ext4 "${TARGET_NVME_DEVICE}"
+sudo mkdir -p "${MOUNT_POINT}"
+# Get device UUID with blkid as exported format:
+# UUID=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+echo "Warning: Will be mounted by UUID in /etc/fstab"
+UUID=$(sudo blkid ${TARGET_NVME_DEVICE} -o export | grep -E "^UUID=")
+echo "${UUID} ${MOUNT_POINT} ext4 noatime 0 0" | sudo tee -a /etc/fstab
+sudo mount -t ext4 -o noatime ${TARGET_NVME_DEVICE} ${MOUNT_POINT}
```
Comment (bryan-bar) [inline edbterraform/data/terraform/gcloud/modules/machine/setup_volume.sh:66]: @jt-edb @mw2q 
Do we expect the default filesystem to be `ext4` or `xfs` ?

Comment (jt-edb) [reply edbterraform/data/terraform/gcloud/modules/machine/setup_volume.sh:66]: xfs

**Review by dougortiz — APPROVED** [review]
Looks good

**Review by jt-edb — APPROVED** [review]
LGTM. Could you please explain in PR's description why we need device name fallback mechanism? Thanks

**Pull request merged**